rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import babel from 'rollup-plugin-babel'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import { uglify } from 'rollup-plugin-uglify'
+
+import pkg from './package.json'
+
+export default {
+  input: 'src/index.js',
+  output: {
+    file: 'dist/bundle.min.js',
+    format: 'cjs',
+    exports: 'auto'
+  },
+  external: [...Object.keys(pkg.dependencies || {})],
```

Comment:
I need to read up on the docs. Would you mind explaining why we need to do this?

Reply:
Hmm.... 🤔

Reply:
Can we pair on this one?